requirements/requirements.txt

```diff
@@ -5,4 +5,4 @@ pandas~=1.1
 pydantic~=1.1,<1.7
 tqdm~=4.23
 toolz~=0.10
-pyarrow==2.0.0
+pyarrow~=2.0
```

Comment:
Version 3 has been released in the mean time.

Reply:
Looking into this PR ~1 year later: version 7 is now there :-)